### DISCUSSION TITLE
Need osgEarth binaries

### DISCUSSION THREAD

**Discussion by BAZI-Ilyess** [pelicanmapping/osgearth#2087] [Support Q&A]
Hy, I'm a new OSGEarth user, Can you send me OSGEarth binaries plzzz, thank you a lot

**Comment by jasonbeverage** [comment]
We don't provide prebuilt binaries but you can use the instructions [here](http://docs.osgearth.org/en/latest/build.html) to simplify building on Windows.  We are working with the vcpkg team to get a PR merged where you will be able to install the latest and greatest osgearth directly from vcpkg.
